.gitignore

```diff
@@ -2,8 +2,11 @@ node_modules
 package-lock.json
 config.js
 client/dist/bundle.js
+<<<<<<< HEAD
```

Comment:
Merge errors like these are potentially what was causing those git errors for you before. Any of the `=======` and `<<<<<` lines can just be deleted